pkg/rest/config_upload.go

```diff
@@ -188,13 +188,21 @@ func getObjectIdIfAlreadyExists(client *http.Client, api api.Api, url string, ob
 		return "", err
 	}
 
+	var found = false
+	var configName = ""
 	for i := 0; i < len(values); i++ {
 		value := values[i]
 		if value.Name == objectName {
-			return value.Id, nil
+			if found {
+				util.Log.Warn("\t\t\tFound two or more configs having same name: %s. Please delete duplicates.", objectName)
+				return configName, nil
```

Comment:
Do we maybe want to return an error here? 
We will only update the first config we find, but if there are multiple,
we cannot be sure it is the one the user desires to update.

Reply:
This would change default behaviour of monaco, causing deployment to fail every time duplicated config is found. Maybe sticking with warning for now and changing behaviour in monaco 2.0 would be a way to go?

Reply:
It changes broken behavior I would argue, but you are right.
[This](https://xkcd.com/1172/) comes to mind :laughing:

Reply:
Follow up ticket to change default behaviour with monaco 2.0: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues/360